tree/treeplayer/inc/ROOT/TTreeProcessorMT.hxx

```diff
@@ -45,7 +47,10 @@ The latter makes a copy of a model object every time a new thread accesses
 the threaded object.
 */
 
+using EntryRange = std::pair<Long64_t, Long64_t>;
```

Comment:
I believe this still pollutes the global namespace of other files including this header right?  Probably better inside the class in any case

Reply:
actually, I am thinking not to expose EntryRange to the user. But they pass an std::pair to TTreeProcessorMT ctor! I think this is the most KISS solution :)

Then we will use this alias (EntryRange) only inside of the internal namespace inside TTreeProcessorMT.cxx (for dev easiness).